### ISSUE TITLE
New rubyzip release 2.0.0

### ISSUE THREAD

**Issue by OpenCoderX** [caxlsx/caxlsx#28]
Looks like there is a setting change that might be more secure:
https://github.com/rubyzip/rubyzip/releases/tag/v2.0.0

**Comment by noniq** [comment]
Looks like this affects extracting ZIP files only. Is this something we even do?

**Comment by OpenCoderX** [comment]
It seems that we do not. There is one reference to `Zip::File.open` but it is commented, maybe someone was going to add functionality to start with an existing file, which would make sense but it is not currently implemented: https://github.com/caxlsx/caxlsx/blob/c1307a3fff72e86074ae19b3d33f79ebebd5e3e6/lib/axlsx/package.rb#L71

**Comment by noniq** [comment]
Thanks for looking into this. I traced this parsing-related code back to a commit in 2011 (3def8f8895). Because it has never been touched since I decided to remove it (bf2cb81), to avoid cluttering searches like this with misleading results.

TL;DR: The change in Rubyzip is only relevant for reading from / extracting ZIP files, which we don’t do, so no immediate action is necessary.
